rust/private/providers.bzl

```diff
@@ -27,6 +27,10 @@ CrateInfo = provider(
         "root": "File: The source File entrypoint to this crate, eg. lib.rs",
         "rustc_env": "Dict[String, String]: Additional `\"key\": \"value\"` environment variables to set for rustc.",
         "srcs": "depset[File]: All source Files that are part of the crate.",
+        "transitive_type": (
```

Comment:
WDYT about naming it "wrapped_crate_type"? If I remember correctly that is how you and Daniel called these rules, "wrapper rules". Or am I misremembering?

Reply:
I'd shy away from "transitive" word though, that usually means everything all the way to leaves, and here we mean something else.

Reply:
I've definitely used the term to describe similar concepts if not this, I can rename